prow/kube/client.go

```diff
@@ -301,6 +306,45 @@ func NewClientFromFile(clusterPath, namespace string) (*Client, error) {
 	return NewClient(&c, namespace)
 }
 
+// TODO: add test that validates that different config formats cannot be confused.
```

Comment:
`// TODO(cjwager): ...` or fix it in this PR.

Reply:
The test cases I added validate this. I forgot to remove this comment. Fixed.